pygen/pygen_src/isa/riscv_cov_instr.py

```diff
@@ -19,8 +19,11 @@
 import logging
 from enum import Enum, IntEnum, auto
 from bitstring import BitArray
-from pygen_src.target.rv32i import riscv_core_setting as rcs
-from pygen_src.riscv_instr_pkg import *
+from pygen.pygen_src.riscv_instr_pkg import *
+if cfg.argv.target == "rv32i":
```

Comment:
@aneels3 and @saurabhsingh-pvips 

Involving `cfg` here needs some additional changes in `riscv_instr_cov_test.py` as I mentioned in #714 in part 2.1 of my comment: 
> 2.1- Since in your implementation, you've involved `cfg` in `riscv_cov_instr.py` to get the correct `rcs`, and because all of the command line argument are parsed in the `riscv_instr_gen_config.py` file, we need to move the command line arguments of `riscv_instr_cov_test.py` to the `riscv_instr_gen_config.py` (similar to what we did for `riscv_instr_base_test.py`). Please note that without this, we will get unrecognized argument error as arguments have been parsed already in the `riscv_instr_gen_config.py` file.

Without making this change, parameters like `--csv_trace` or `--enable_visualization` won't be recognized.